.github/workflows/deploy-development-docker-image.yml

```
@@ -0,0 +1,44 @@
+name: Create and publish development Docker image
+
+on:
+  push:
+    branches-ignore:
+      - "main"
```

if its the development docker image, why ignore main, and not just on development branch? now it creates a new image on every branch push no?

```suggestion
    branches:
      - "development"
```

---

I discarded the idea of a development branch and instead created an action that tags a docker image with `development` on every branch except main. So while development the development image is released on every pushed commit.
This will be a problem if we develop on two branches in parallel but is easier to handle than merging every time in an extra development branch. If this is a problem we can simply disable the action for the branch not needing the development deployment.

---

Hm i See, i actually dont know what is better, so lets let it be like you decidet because its less merging and branch spamming.